packages/verification-vue-component/src/components/Verification/items/cards/FaqCard.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <BaseCard :card-class="'faq-card'">
+        <template #header>
+            <div class="title">
+                <span class="help-card-title">{{ _$t('verification.card.faq.title') }}</span>
+            </div>
+        </template>
+        <template #body>
+            <div class="faq-items">
+                <div class="item"
+                     v-for="(item, index) in _$t('verification.card.faq.items', { returnObjects: true })"
+                     :key="index">
+                    <input type="checkbox" :id="'chck' + index">
```

Comment:
Why using this workaround in vuejs? Also the open/close animation looks quite strange, because you're using the max-height method which. For credittrust I've created this component which uses the actual height when opened for the animation: https://github.com/certifaction/credittrust/blob/development/frontend/src/components/Accordion.vue
I think it would be easy to reuse this component here.

Reply:
Using the one from credittrust now